### ISSUE TITLE
Total upload progress

### ISSUE THREAD

**Issue by tbertels** [ritschwumm/commonist#17]
While there's an upload progress for the current file, there isn't any for the list of selected files. 

A simple "File X/Total" would be useful, but a progress bar (based on total size to upload) would be even better if possible.

**Comment by ritschwumm** [comment]
two progress bars might be a bit of overkill - and doesn't fit well with displaying the message in the same place. what do you think about _only_ displaying the total progress? in the end i think that's the only thing you really want to know...

**Comment by tbertels** [comment]
Indeed, one total progress bar - ideally progressing even during the current file upload - would be better.
